mises/chain.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mises",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mises.site",
+  "pretty_name": "Mises",
+  "chain_id": "mainnet",
+  "bech32_prefix": "mis",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Registered and fixed typo from `mis` to `mises`
https://github.com/satoshilabs/slips/commit/1cbb46e974d306d512f16dbe6cfa23dffa86990b
https://github.com/cosmos/chain-registry/pull/1068/commits/94a6b5ca29dacaa998da5aa611f3342e97a3e7e9